verl/trainer/ppo/ray_trainer.py

```diff
@@ -504,6 +504,7 @@ def check_mutually_exclusive(mbs, mbs_per_gpu, name: str):
         #    ppo_mini_batch_size is divisible by ppo_micro_batch_size
         #    ppo_micro_batch_size * sequence_parallel_size >= n_gpus
         if not config.actor_rollout_ref.actor.use_dynamic_bsz:
+            assert config.actor_rollout_ref.train_batch_size % config.actor_rollout_ref.actor.ppo_mini_batch_size == 0
```

Comment:
It's not `config.actor_rollout_ref.train_batch_size` but `config.data.train_batch_size`.
Also, this assertion would be useful for the critic model.
Could you help adding it for critic?